pykanidm/pyproject.toml

```diff
@@ -81,3 +81,6 @@ omit = ["tests"]
 
 [tool.mypy]
 plugins = "pydantic.mypy"
+
+[tool.black]
```

Comment:
Please don't use black, it'll argue with `ruff` which is part of the test suite.